firefox-ios/Client/Frontend/Autofill/Address/AddressListViewModel.swift

```diff
@@ -68,6 +70,31 @@ class AddressListViewModel: ObservableObject, FeatureFlaggable {
         self.logger = logger
         self.windowUUID = windowUUID
         self.addressProvider = addressProvider
+        preloadWebView()
+    }
+
+    func preloadWebView() {
+        // Reuse existing webview if it exists
+        guard webView == nil else { return }
+        let webConfiguration = WKWebViewConfiguration()
+        webView = WKWebView(frame: .zero, configuration: webConfiguration)
+        webView?.translatesAutoresizingMaskIntoConstraints = false
+
+        // Allow Safari Web Inspector (requires toggle in Settings > Safari > Advanced).
+        if #available(iOS 16.4, *) {
+            webView?.isInspectable = true
+        }
```

Comment:
Would this also be part of the View considering webViews are views? cc @razvanlitianu @issammani ?

Reply:
Hmm it makes sense to have it in the view as well.

Reply:
Awaiting on @razvanlitianu before we merge this in

Reply:
As of https://github.com/mozilla-mobile/firefox-ios/pull/20738/commits/38cfe42fb9843e9bdcaeecf5256f3204e558e954 this is now part of the webview manager. I also added a check to only enable this for simulator. Not sure if that's overkill or not. What do you think @nbhasin2 @razvanlitianu ?

https://github.com/mozilla-mobile/firefox-ios/blob/38cfe42fb9843e9bdcaeecf5256f3204e558e954/firefox-ios/Client/Frontend/Autofill/Address/Edit/EditAddressWebViewManager.swift#L21-L28